Mathlib/Topology/Algebra/MvPolynomial.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2023 Ruben Van de Velde. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde
+-/
+
+import Mathlib.Data.MvPolynomial.Basic
+import Mathlib.Topology.Algebra.Ring.Basic
+
+/-!
+# Multivariate polynomials and continuity
+
+In this file we prove the following lemma:
+
+* `MvPolynomial.continuous_eval`: `MvPolynomial.eval` is continuous
+
+## Tags
+
+MvPolynomial, continuity
```

Comment:
The point of tags is to help discoverability, not to parrot what's already in the file
```suggestion
multi-variate polynomial, continuity
```